src/sentry/static/sentry/app/views/organizationIntegrations/integrationItem.tsx

```diff
@@ -42,7 +47,7 @@ export default class IntegrationItem extends React.Component {
   }
 }
 
-const Labels = styled('div')`
+const Labels = styled('div')<Pick<Props, 'compact'>>`
```

Comment:
You have this twice, should you make a separate type definition elsewhere in the module?